package.json

```diff
@@ -29,6 +29,8 @@
     "@auth0/angular-jwt": "~5.1.0",
     "@swimlane/ngx-charts": "~20.1.2",
     "@types/file-saver": "^2.0.5",
+    "angular-star-rating": "^6.0.0",
+    "css-star-rating": "^1.3.1",
```

Comment:
This is required by angular star rating or is it an older version that has been abandoned (and should be removed as a result from package.json)?